js/tictactoe.js

```diff
@@ -0,0 +1,229 @@
+		document.getElementById("square5").innerHTML = playerTwoMove;
+		gameStatus.innerHTML = playerOneTurn;
+		playerTurn++;
+		document.getElementById("square5").style.backgroundColor = oCellColor;
+	}
+	checkForWinner();	
+});
+
+document.getElementById("square6").addEventListener("click", function() {
+	if (playerTurn % 2 != 0 && document.getElementById("square6").innerHTML === "") {
+		document.getElementById("square6").innerHTML = playerOneMove;
+		gameStatus.innerHTML = playerTwoTurn;
+		playerTurn++;
+		document.getElementById("square6").style.backgroundColor = xCellColor;
+	}
+	else if (playerTurn % 2 === 0 && document.getElementById("square6").innerHTML === ""){
+		document.getElementById("square6").innerHTML = playerTwoMove;
+		gameStatus.innerHTML = playerOneTurn;
+		playerTurn++;
+		document.getElementById("square6").style.backgroundColor = oCellColor;
+	}
+	checkForWinner();	
+});
+
+document.getElementById("square7").addEventListener("click", function() {
+	if (playerTurn % 2 != 0 && document.getElementById("square7").innerHTML === "") {
+		document.getElementById("square7").innerHTML = playerOneMove;
+		gameStatus.innerHTML = playerTwoTurn;
+		playerTurn++;
+		document.getElementById("square7").style.backgroundColor = xCellColor;
+	}
+	else if (playerTurn % 2 === 0 && document.getElementById("square7").innerHTML === ""){
+		document.getElementById("square7").innerHTML = playerTwoMove;
+		gameStatus.innerHTML = playerOneTurn;
+		playerTurn++;
+		document.getElementById("square7").style.backgroundColor = oCellColor;
+	}
+	checkForWinner();	
+});
+
+document.getElementById("square8").addEventListener("click", function() {
+	if (playerTurn % 2 != 0 && document.getElementById("square8").innerHTML === "") {
+		document.getElementById("square8").innerHTML = playerOneMove;
+		gameStatus.innerHTML = playerTwoTurn;
+		playerTurn++;
+		document.getElementById("square8").style.backgroundColor = xCellColor;
+	}
+	else if (playerTurn % 2 === 0 && document.getElementById("square8").innerHTML === ""){
+		document.getElementById("square8").innerHTML = playerTwoMove;
+		gameStatus.innerHTML = playerOneTurn;
+		playerTurn++;
+		document.getElementById("square8").style.backgroundColor = oCellColor;
+		}
+		checkForWinner();	
+});
+
+document.getElementById("square9").addEventListener("click", function() {
+	if (playerTurn % 2 != 0 && document.getElementById("square9").innerHTML === "") {
+		document.getElementById("square9").innerHTML = playerOneMove;
+		gameStatus.innerHTML = playerTwoTurn;
+		playerTurn++;
+		document.getElementById("square9").style.backgroundColor = xCellColor;
+	}
+	else if (playerTurn % 2 === 0 && document.getElementById("square9").innerHTML === ""){
+		document.getElementById("square9").innerHTML = playerTwoMove;
+		gameStatus.innerHTML = playerOneTurn;
+		playerTurn++;
+		document.getElementById("square9").style.backgroundColor = oCellColor;
+	}
+	checkForWinner();
```

Comment:
The logic looks sound for these square listeners, awesome! Keep in mind how each function is very similar. This would be a great case for using a function, such as this one:

```js
function squareClick(squareName) {
  if (playerTurn % 2 != 0 && document.getElementById(squareName).innerHTML === "") {
    //code here
  } else if (playerTurn % 2 === 0 && document.getElementById(squareName).innerHTML === ""){
    //code here
  }
  checkForWinner();
}
```

Then, each event listener could look more like so:

```js
document.getElementById("square9").addEventListener("click", function() {
  squareClick("square9");
});
```

This will also help if you want to make a change to that code. Instead of altering all 9 events, you can just edit the function.